Test/spec/simd/simd_load.wast

```diff
@@ -0,0 +1,180 @@
+;; v128.load operater with normal argument (e.g. (i8x16, i16x8 i32x4))
+
+(module
+  (memory 1)
+  (func (export "v128.load") (param $address v128) (result v128)
+    (local v128)
+    (v128.store (i32.const 0) (local.get $address))
+    (v128.load (i32.const 0))
+  )
+)
+
+(assert_return (invoke "v128.load" (v128.const i8x16 0 1 2 3 4 5 6 7 8 9 10 11 12 13 14 15)) (v128.const i8x16 0 1 2 3 4 5 6 7 8 9 10 11 12 13 14 15))
+(assert_return (invoke "v128.load" (v128.const i16x8 0 1 2 3 4 5 6 7)) (v128.const i16x8 0 1 2 3 4 5 6 7))
+(assert_return (invoke "v128.load" (v128.const i32x4 0 1 2 3)) (v128.const i32x4 0 1 2 3))
+
+
+;; v128.load operater as the argument of other SIMD instructions
+
+(module (memory 1)
+  (data (i32.const 0) "abcdefghijklmnopqrstuvwxyzaBCdEfabIjklmnop")
+  (func (export "as-i8x16_extract_lane_s-value/0") (result i32)
+    (i8x16.extract_lane_s 0 (v128.load (i32.const 0)))
+  )
+  (func (export "as-i8x16.eq-operand") (result v128) (i8x16.eq (v128.load offset=0 (i32.const 0)) (v128.load offset=26 (i32.const 0))))
+  (func (export "as-v128.not-operand") (param $0 v128) (result v128)
+    (local v128)
+    (v128.store (i32.const 0) (local.get $0))
+    (v128.not (v128.load (i32.const 0)))
```

Comment:
I think these tests would be better if they didn't store to the address in the same function as the load. WAVM doesn't make this optimization, but I believe VMs are allowed to eliminate the load like this:
```
(v128.store (i32.const 0) (local.get $0))
(v128.not (local.get $0))
```

Reply:
I see your point, but on the other hand such an engine could get value from these tests with the optimizations enabled or disabled, testing both the optimization and the execution.

Reply:
Maybe it would be useful to add a separate test (or tests) for store->load forwarding? I think these tests are useful without it, because they test generating X86 memory operands.

Reply:
My concern is, it's not easy to evaluate the optimization is enabled or not for a spec test, even it's out of the scope of the spec testing. Spec tests only ensure implementations are compliance with spec. 

But I'm OK to remove the store operator here to avoid potential optimization may affect what these test cases originally wanted to test.